common/file.cpp

```diff
@@ -219,4 +219,14 @@ bool DumpFile::flush() {
 
 int32 DumpFile::pos() const { return _handle->pos(); }
 
+bool DumpFile::seek(int32 offset, int whence) {
+	SeekableWriteStream *ws = dynamic_cast<SeekableWriteStream *>(_handle);
+	return ws ? ws->seek(offset, whence) : -1;
```

Comment:
`-1` should be `false` here.

Reply:
Thanks. I've committed a fix for the problems